src/database-postgresql.h

```diff
@@ -27,69 +27,123 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 class Settings;
 
-class Database_PostgreSQL : public Database
+class Database_PostgreSQL: public Database
```

Comment:
you have Database for non roled databases, MapDatabase for maps, and PlayerDatabase for players